pkg/servicecentral/client.go

```diff
@@ -344,7 +338,8 @@ func (c *Client) DeleteService(ctx context.Context, user auth.User, serviceUUID
 }
 
 // GetServiceAttributes queries service central for the attributes of a given service. Can return an empty array if no attributes were found
-func (c *Client) GetServiceAttributes(ctx context.Context, user auth.OptionalUser, serviceUUID string) ([]ServiceAttributeResponse, error) {
+func (c *Client) GetServiceAttributes(ctx context.Context, user auth.OptionalUser, serviceUUID string) ([]ServiceAttribute, error) {
+	var svcAttrResp []ServiceAttribute
```

Comment:
What is the reason for this change? Returning `nil` is the idiomatic thing to do - it is immediately clear that the value is not supposed to have anything in it and should not be used. With this change it is ambiguous - maybe the slice is empty or maybe not? You are forced to read the code to see it's empty.